xrootd/client/protocol.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package client // import "go-hep.org/x/hep/xrootd/client"
+
+import (
+	"context"
+
+	"go-hep.org/x/hep/xrootd/encoder"
+	"go-hep.org/x/hep/xrootd/protocol"
+	protocolRequest "go-hep.org/x/hep/xrootd/protocol/protocol"
```

Comment:
package names are usually not "camel-cased".
could we think about a better name in this unfortunate name collision?

`xproto` ? `protoreq` ?

Reply:
or name the other import like:
```go
import (
    xrdproto "go-hep.org/x/hep/xrootd/protocol"
    "go-hep.org/x/hep/xrootd/protocol/protocol"
)
```
?